.github/workflows/ci.yml

```diff
@@ -56,6 +56,12 @@ jobs:
       - run: |
           python scripts/doc_checker.py --add-ignore=D101,D102 --disable-numpydoc \
             modin/pandas/series_utils.py
+      - run: |
+          python scripts/doc_checker.py --add-ignore=D102 --disable-numpydoc \
```

Comment:
Can you enable numpydoc check?

Reply:
There is a problem with `numpydoc` in `modin/engines/base/io/io.py` because in this module `@doc` decorator was used for functions documenting and `numpydoc` doesn't recognize it.

Reply:
Could you add details? As far as I can tell, the problem is that the added documentation lacks a description of the parameters.

Reply:
Oh, sorry, i misled you, `--disable-numpydoc` was added because docstrings in `modin/engines/base/io/io.py` don't have parameters sections, docstring just refers to pandas API, for example
```
python -c "from modin.engines.base.io.io import BaseIO; print(BaseIO.read_csv.__doc__)"

Read a comma-separated values (CSV) file into query compiler using pandas.

For parameters description please refer to pandas API.

Returns
-------
BaseQueryCompiler or TextParser
    QueryCompiler or TextParser with read data.


See `Pandas API documentation <https://pandas.pydata.org/pandas-docs/version/1.2.3/reference/api/pandas.read_csv.html>`_ for more.
```

And then, if we run `doc-checker` without `--disable-numpydoc`, we will get the next error message
`python scripts/doc_checker.py --add-ignore=D102  modin/engines/base/io/io.py`
<details>
  <summary>
    Output
  </summary>

```INFO:NUMPYDOC OUTPUT FOR modin\engines\base\io\io.py
ERROR:modin.engines.base.io.io.BaseIO.read_parquet:99:PR01:Parameters {'engine', 'use_nullable_dtypes', '**kwargs', 'columns', 'path'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_csv:112:PR01:Parameters {'names', 'iterator', 'encoding', 'skip_blank_lines', 'infer_datetime_format', 'quotechar', 'float_precision', 'verbose', 'filepath_or_buffer', 'warn_bad_lines', 'dayfirst', 'dtype', 'keep_date_col', 'delim_whitespace', 'error_bad_lines', 'quoting', 'delimiter', 'chunksize', 'low_memory', 'prefix', 'memory_map', 'converters', 'nrows', 'engine', 'false_values', 'parse_dates', 'skiprows', 'cache_dates', 'compression', 'mangle_dupe_cols', 'thousands', 'dialect', 'lineterminator', 'doublequote', 'na_filter', 'storage_options', 'comment', 'header', 'sep', 'keep_default_na', 'na_values', 'usecols', 'index_col', 'escapechar', 'true_values', 'skipfooter', 'squeeze', 'skipinitialspace', 'date_parser', 'decimal'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_json:254:PR01:Parameters {'date_unit', 'encoding', 'lines', 'path_or_buf', 'dtype', 'orient', 'numpy', 'storage_options', 'typ', 'chunksize', 'keep_default_dates', 'nrows', 'convert_axes', 'convert_dates', 'precise_float', 'compression'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_gbq:301:PR01:Parameters {'configuration', 'dialect', 'query', 'project_id', 'auth_local_webserver', 'credentials', 'verbose', 'index_col', 'max_results', 'reauth', 'location', 
'private_key', 'progress_bar_type', 'use_bqstorage_api', 'col_order'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_html:347:PR01:Parameters {'attrs', 'keep_default_na', 'thousands', 'na_values', 'encoding', 'index_col', 'flavor', 'match', 'io', 'converters', 'parse_dates', 'skiprows', 'displayed_only', 'header', 'decimal'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_clipboard:392:PR01:Parameters {'sep', '**kwargs'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_excel:403:PR01:Parameters {'names', 'mangle_dupe_cols', 'thousands', 'verbose', 'io', 'dtype', 'na_filter', 'comment', 'header', 'skip_footer', 'keep_default_na', 'na_values', 
'sheet_name', '**kwds', 'usecols', 'converters', 'nrows', 'engine', 'index_col', 'true_values', 'skipfooter', 'false_values', 'squeeze', 'date_parser', 'parse_dates', 'skiprows', 'convert_float'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_hdf:480:PR01:Parameters {'mode', 'iterator', 'chunksize', 'columns', 'start', 'where', 'path_or_buf', '**kwargs', 'stop', 'errors', 'key'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_feather:518:PR01:Parameters {'use_threads', 'storage_options', 'columns', 'path'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_stata:536:PR01:Parameters {'iterator', 'chunksize', 'order_categoricals', 'preserve_dtypes', 'convert_missing', 'convert_categoricals', 'filepath_or_buffer', 'index_col', 'convert_dates', 'storage_options', 'columns'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_sas:573:PR01:Parameters {'iterator', 'chunksize', 'encoding', 'index', 'format', 'filepath_or_buffer'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_pickle:601:PR01:Parameters {'storage_options', 'filepath_or_buffer', 'compression'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_sql:618:PR01:Parameters {'chunksize', 'index_col', 'coerce_float', 'con', 'params', 'parse_dates', 'sql', 'columns'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_fwf:650:PR01:Parameters {'infer_nrows', 'widths', '**kwds', 'filepath_or_buffer', 'colspecs'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_sql_table:679:PR01:Parameters {'chunksize', 'schema', 'index_col', 'coerce_float', 'con', 'parse_dates', 'table_name', 'columns'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_sql_query:711:PR01:Parameters {'coerce_float', 'con', 'params', 'chunksize', 'parse_dates', 'sql', 'index_col'} not documented
ERROR:modin.engines.base.io.io.BaseIO.read_spss:741:PR01:Parameters {'convert_categoricals', 'usecols', 'path'} not documented
ERROR:modin.engines.base.io.io.BaseIO.to_sql:752:PR01:Parameters {'chunksize', 'schema', 'if_exists', 'index', 'qc', 'dtype', 'con', 'method', 'index_label', 'name'} not documented
ERROR:modin.engines.base.io.io.BaseIO.to_pickle:785:PR01:Parameters {'obj', 'protocol', 'compression', 'path'} not documented
ERROR:modin.engines.base.io.io.BaseIO.to_csv:804:PR01:Parameters {'obj', '**kwargs'} not documented
ERROR:INVALID DOCUMENTATION FOUND
```
</details>

And adding `PR01` code to the `add-ignore` list gives the same result.

Reply:
Try add code in the way:
```python
def read_parquet(cls, path, engine, columns, use_nullable_dtypes, **kwargs):  # noqa: PR01
```

Reply:
`add-ignore` only work for pydocstyle codes

Reply:
Ok, i see. `noqa` statements were added.